Tuto-GUDHI-persistent-entropy.ipynb

```diff
@@ -0,0 +1,823 @@
+{
```

Comment:
<p>Typo, "with respect to"</p><p>I also wonder if Betti should be capitalized (throughout the document). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>Typo fixed,</p><p>and yes, I think it should be capitalized (see for example https://en.wikipedia.org/wiki/Betti_number).</p><p>I am doing it.</p><p> </p>